modules/acquia_cms_site_studio/acquia_cms_site_studio.install

```diff
@@ -448,3 +412,23 @@ function acquia_cms_site_studio_update_8009() {
     }
   }
 }
+
+/**
+ * Install Node Revision Delete module with default configuration.
+ *
+ * Implements hook_update_n()
+ */
+function acquia_cms_site_studio_update_9000() {
+  // Install node revision delete module and set default configurations.
+  \Drupal::service('module_installer')->install(['node_revision_delete']);
+  $config = \Drupal::service('config.factory')->getEditable('node_revision_delete.settings');
+  if ($config) {
+    $config->set('defaults.amount.status', TRUE)
+      ->set('defaults.amount.settings.amount', 50)
+      ->set('defaults.created.status', TRUE)
+      ->set('defaults.created.settings.age', 12)
+      ->set('defaults.drafts.status', TRUE)
+      ->set('defaults.drafts.settings.age', 12)
+      ->save();
+  }
```

Comment:
```suggestion
  \Drupal::service('module_installer')->install(['node_revision_delete']);
  $config = \Drupal::service('config.factory')->getEditable('node_revision_delete.settings');
  if ($config) {
    $config->set('defaults.amount.status', TRUE)
      ->set('defaults.amount.settings.amount', 50)
      ->set('defaults.created.status', TRUE)
      ->set('defaults.created.settings.age', 12)
      ->set('defaults.drafts.status', TRUE)
      ->set('defaults.drafts.settings.age', 12)
      ->save();
  }
```
I think we can call this section once in a method.

Reply:
Done